src/MongoDB/Manager.c

```diff
@@ -773,30 +783,28 @@ static phongo_create_object_retval php_phongo_manager_create_object(zend_class_e
 #else
 	{
 		zend_object_value retval;
-		retval.handle = zend_objects_store_put(intern, (zend_objects_store_dtor_t) zend_objects_destroy_object, php_phongo_manager_free_object, NULL TSRMLS_CC);
+		retval.handle   = zend_objects_store_put(intern, (zend_objects_store_dtor_t) zend_objects_destroy_object, php_phongo_manager_free_object, NULL TSRMLS_CC);
 		retval.handlers = &php_phongo_handler_manager;
 
 		return retval;
 	}
 #endif
 } /* }}} */
 
-static HashTable *php_phongo_manager_get_debug_info(zval *object, int *is_temp TSRMLS_DC) /* {{{ */
+static HashTable* php_phongo_manager_get_debug_info(zval* object, int* is_temp TSRMLS_DC) /* {{{ */
 {
-	php_phongo_manager_t         *intern;
-	mongoc_server_description_t **sds;
+	php_phongo_manager_t*         intern;
+	mongoc_server_description_t** sds;
 	size_t                        i, n = 0;
+	zval                          retval = ZVAL_STATIC_INIT;
 #if PHP_VERSION_ID >= 70000
-	zval                          retval, cluster;
+	zval cluster;
```

Comment:
Can we rewrite this as `ZVAL_RETVAL_TYPE cluster`? That removes the `NULL` assignment for PHP 5.x, but I don't think we need to worry about that since we'll pass it to `MAKE_STD_ZVAL()` anyway.